playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/upgrade.yml

```diff
@@ -1,5 +1,19 @@
 ---
+###############################################################################
+# The restart playbook should be run after this playbook completes.
+###############################################################################
+
+- name: Upgrade docker
+  hosts: oo_masters_to_config:oo_nodes_to_config:oo_etcd_to_config
+  roles:
+  - openshift_facts
+  tasks:
+  - include: docker_upgrade.yml
```

Comment:
That file already existed.  I just had to move it earlier due to bugs in docker 1.8.2's --add-registry.  Without docker 1.9 the workaround for authenticated registries doesn't even work.